bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/CheckAndMutateUtil.java

```diff
@@ -62,20 +63,20 @@ public void applyPreSendHook(Query query) {
   /**
    * <p>wasMutationApplied.</p>
    *
-   * @param request a {@link com.google.bigtable.v2.CheckAndMutateRowRequest} object.
-   * @param response a {@link com.google.bigtable.v2.CheckAndMutateRowResponse} object.
+   * @param request a {@link ConditionalRowMutation} object.
+   * @param predicateMatched a {@link Boolean} object.
    * @return a boolean.
    */
   public static boolean wasMutationApplied(
-      CheckAndMutateRowRequest request,
-      CheckAndMutateRowResponse response) {
+      ConditionalRowMutation request,
+      Boolean predicateMatched) {
 
+    CheckAndMutateRowRequest proto =
+        request.toProto(RequestContext.create("SomeProject", "Some Instance", ""));
```

Comment:
Should we add hasTrueMutations() and hasFalseMutations() to the veneer model?
Or maybe checkAndMutateRow should return an enum with 3 possible outcomes:  true mutations applied, false mutations applied, nothing happened?

Reply:
I think that new `has()` methods in the model would be ideal.